CMakeLists.txt

```diff
@@ -509,6 +509,7 @@ target_sources(OpenSTA
 )
 
 target_link_libraries(OpenSTA
+  stdc++fs
```

Comment:
this has to be conditionalized because it cause errors:
ld: library 'stdc++fs' not found
clang: error: linker command failed with exit code 1 (use -v to see invocation)

Reply:
I'll fix